lib/game_logic.rb

```diff
@@ -0,0 +1,89 @@
+class TicTacToe
```

Comment:
Nice job encapsulating this logic within a CLI class!! You'll usually want to give this file the same name as the class (except in snake case) and will likely want to name the class itself to indicate its role as the CLI.